src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -772,7 +774,69 @@ public String getFileType(final Map<String, String> metaData, final String forms
     }
 
     /**
-     * Parameters that are used to determine filetype in {@link #getFileType(Map, String)}
+     * Get the file type from the metadata or the form string passed in
```

Comment:
Update method description by replacing metadata and form string with IBaseDataObject